app/initializers/ember-cli-mirage.js

```diff
@@ -23,7 +25,7 @@ export function startMirage(env = ENV) {
   let modules = readModules(env.modulePrefix);
   let options = _assign(modules, {environment, baseConfig, testConfig});
 
-  return new Server(options);
+  return new Server(_assign({createInterceptor: createPretender}, options));
```

Comment:
Don't think we need this or either of the `create*` functions here, the `createPretender` is already a default option within `Server`.